 Network rinha-nginx-2024q1  Creating
 Network rinha-nginx-2024q1  Created
 Container exemplo-db-1  Creating
 Container exemplo-db-1  Created
 Container exemplo-api02-1  Creating
 Container exemplo-api01-1  Creating
 Container exemplo-api01-1  Created
 Container exemplo-api02-1  Created
 Container exemplo-nginx-1  Creating
 Container exemplo-nginx-1  Created
Attaching to api01-1, api02-1, db-1, nginx-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
db-1     | selecting default time zone ... Etc/UTC
db-1     | creating configuration files ... ok
db-1     | running bootstrap script ... ok
db-1     | performing post-bootstrap initialization ... ok
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | waiting for server to start....2024-02-11 06:06:03.339 UTC [48] LOG:  starting PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-11 06:06:03.340 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-11 06:06:03.345 UTC [51] LOG:  database system was shut down at 2024-02-11 06:06:03 UTC
db-1     | 2024-02-11 06:06:03.353 UTC [48] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
nginx-1  | 2024/02/11 06:06:03 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.30.0.4:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/11 06:06:03 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.30.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.30.0.3:3000/clientes/1/extrato", host: "localhost:9999"
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | DO
db-1     | CREATE FUNCTION
db-1     | CREATE FUNCTION
db-1     | 
db-1     | 
db-1     | 2024-02-11 06:06:03.710 UTC [48] LOG:  received fast shutdown request
db-1     | 2024-02-11 06:06:03.710 UTC [48] LOG:  aborting any active transactions
db-1     | waiting for server to shut down....2024-02-11 06:06:03.716 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
db-1     | 2024-02-11 06:06:03.719 UTC [49] LOG:  shutting down
db-1     | 2024-02-11 06:06:03.719 UTC [49] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-11 06:06:03.732 UTC [49] LOG:  checkpoint complete: wrote 946 buffers (5.8%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.013 s, sync=0.001 s, total=0.013 s; sync files=0, longest=0.000 s, average=0.000 s; distance=4314 kB, estimate=4314 kB; lsn=0/19212F8, redo lsn=0/19212F8
db-1     | 2024-02-11 06:06:03.744 UTC [48] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-02-11 06:06:03.856 UTC [1] LOG:  starting PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-11 06:06:03.859 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-02-11 06:06:03.859 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-02-11 06:06:03.859 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-11 06:06:03.863 UTC [66] LOG:  database system was shut down at 2024-02-11 06:06:03 UTC
db-1     | 2024-02-11 06:06:03.873 UTC [1] LOG:  database system is ready to accept connections
api01-1  | 2024-02-11 06:06:04.358:INFO::main: Logging initialized @2627ms
api02-1  | 2024-02-11 06:06:04.399:INFO::main: Logging initialized @2663ms
nginx-1  | 2024/02/11 06:06:05 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 172.30.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/11 06:06:07 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 172.30.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api02-1  | Feb 11, 2024 6:06:08 AM com.mchange.v2.log.MLog 
api02-1  | INFO: MLog clients using java 1.4+ standard logging.
api01-1  | Feb 11, 2024 6:06:09 AM com.mchange.v2.log.MLog 
api01-1  | INFO: MLog clients using java 1.4+ standard logging.
api01-1  | Feb 11, 2024 6:06:09 AM com.mchange.v2.c3p0.C3P0Registry 
api01-1  | INFO: Initializing c3p0-0.9.5.4 [built 23-March-2019 23:00:48 -0700; debug? true; trace: 10]
api02-1  | Feb 11, 2024 6:06:09 AM com.mchange.v2.c3p0.C3P0Registry 
api02-1  | INFO: Initializing c3p0-0.9.5.4 [built 23-March-2019 23:00:48 -0700; debug? true; trace: 10]
api01-1  | 2024-02-11 06:06:09.449:INFO:oejs.Server:main: jetty-9.2.z-SNAPSHOT
nginx-1  | 2024/02/11 06:06:09 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 172.30.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api01-1  | 2024-02-11 06:06:09.508:INFO:oejs.ServerConnector:main: Started ServerConnector@5776a9f8{HTTP/1.1}{0.0.0.0:3000}
api01-1  | 2024-02-11 06:06:09.508:INFO:oejs.Server:main: Started @7777ms
api01-1  | Started server on port 3000
api02-1  | 2024-02-11 06:06:09.537:INFO:oejs.Server:main: jetty-9.2.z-SNAPSHOT
api02-1  | 2024-02-11 06:06:09.591:INFO:oejs.ServerConnector:main: Started ServerConnector@60377eb8{HTTP/1.1}{0.0.0.0:3000}
api02-1  | 2024-02-11 06:06:09.592:INFO:oejs.Server:main: Started @7856ms
api02-1  | Started server on port 3000
nginx-1  | 2024/02/11 06:06:11 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 172.30.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/11 06:06:13 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 172.30.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
api02-1  | Feb 11, 2024 6:06:15 AM com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource 
api02-1  | INFO: Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1br8av7b115u13a714bqw7m|4a3404fa, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> org.postgresql.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 1br8av7b115u13a714bqw7m|4a3404fa, idleConnectionTestPeriod -> 800, initialPoolSize -> 13, jdbcUrl -> jdbc:postgresql://db:5432/rinha?ApplicationName=rinha-web-server, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 3600, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 120, maxPoolSize -> 13, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 3, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
api01-1  | Feb 11, 2024 6:06:16 AM com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource 
api01-1  | INFO: Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 3, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1br8av8b115u138zfi37g5|2b5a04b0, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> org.postgresql.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 1br8av8b115u138zfi37g5|2b5a04b0, idleConnectionTestPeriod -> 800, initialPoolSize -> 13, jdbcUrl -> jdbc:postgresql://db:5432/rinha?ApplicationName=rinha-web-server, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 3600, maxIdleTime -> 1800, maxIdleTimeExcessConnections -> 120, maxPoolSize -> 13, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 3, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
nginx-1 exited with code 0
nginx-1 exited with code 0
api02-1 exited with code 0
api02-1 exited with code 143
api01-1 exited with code 0
api01-1 exited with code 143
db-1     | 2024-02-11 06:10:32.400 UTC [1] LOG:  received fast shutdown request
db-1     | 2024-02-11 06:10:32.400 UTC [1] LOG:  aborting any active transactions
db-1     | 2024-02-11 06:10:32.403 UTC [1] LOG:  background worker "logical replication launcher" (PID 69) exited with exit code 1
db-1     | 2024-02-11 06:10:32.403 UTC [64] LOG:  shutting down
db-1     | 2024-02-11 06:10:32.404 UTC [64] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-11 06:10:32.408 UTC [64] LOG:  checkpoint complete: wrote 610 buffers (3.7%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.005 s, sync=0.001 s, total=0.005 s; sync files=0, longest=0.000 s, average=0.000 s; distance=1927 kB, estimate=1927 kB; lsn=0/1B03178, redo lsn=0/1B03178
db-1     | 2024-02-11 06:10:32.415 UTC [1] LOG:  database system is shut down
db-1 exited with code 0
